### ISSUE TITLE
Word table selection keys aren't read

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#5961]
Aside from NVDA's table navigation keys (control+alt+arrows), Word includes some of its own:

alt+home (move to first cell in row)
alt+end (move to first last in row)
alt+page up (move to first cell in column)
alt+page down (move to last cell in column)

Plus, add SHIFT to select (eg alt+home then alt+shift+end to select current row)

also alt+shift+up to select the row above and alt+shift+down arrow to select the row below (I probably wouldn't recommend these as if you are already in the top or bottom row they actually separate that row from the rest of the table - which is arguably even more important to know when you've done it).

Pressing any of these keys results in the cursor moving as expected, but NVDA does not announce anything.

**Comment by nvaccessAuto** [comment]
Incubated in dd7013c60924e26425c8cece0fcaad04e0741baf.

**Comment by nvaccessAuto** [comment]
Incubated in 8af1ac0d3347dc6d995864fbdc934e9b380e466f.

**Comment by michaelDCurran** [comment]
Due to a deficiency in the MS Word object model, alt+shift+pageUp and alt+shift+pageDown will read more text than they should when reading the selection. I.e. If selecting down the middle column of a 3 column table, it will also include row 1 column 3, row 2 column 1, row 2 column 3 and row 3 column 1, as the MS Word object model cannot give back noncontiguous ranges. In the long-term we may consider some kind of special selection reading code for MS Word tables, but as this affects not just these particular commands, we should worry about it in another issue.
